lib/tasks/clear_vocab_sheets.rake

```diff
@@ -0,0 +1,12 @@
+namespace :vocab_sheets do
+  MAX_DAYS = 31
+
+  desc 'Clears all vocab sheets that are more than one month old'
+  task clear: :environment do
+    total_destroyed = 0
+    total_destroyed += VocabSheet.where('created_at < ?', MAX_DAYS.days.ago).destroy_all.length
```

Comment:
Might be smarter to change this to 'updated_at < ?' ?

Reply:
is this was a class method on the VocabSheet model instead, you could write some nice tests.